app/models/user.rb

```diff
@@ -12,6 +12,8 @@ class User < ApplicationRecord
 
   validates :gender, :age, presence: true
 
+  scope :young_women, -> { where('age between ? and ? and gender = ?', 20, 30, 1) }
```

Comment:
'1'? I know that this is women but someone else, no :) Could you use enum value here?